changelog.md

```diff
@@ -1,3 +1,12 @@
+# 3.11.0
```

Comment:
Because of our `~`?
Probably best to just use `Next` untill we actually cut a release.